app/src/main/res/layout/content_main.xml

```diff
@@ -145,9 +148,9 @@
             android:hint="Samples per second"
             android:inputType="number"
             android:selectAllOnFocus="true"
-            android:text="2400000"
```

Comment:
Removed the defaults specified in the UI stuff because they're not representative of the java side